### ISSUE TITLE
Problem in sub-repeater and dotcontrol

### ISSUE THREAD

**Issue by Compactis** [riganti/dotvvm#531]
When I click on the first buttons of all collections, the event is called correctly, but others report an error: System.Exception: Unauthorized call invocation!
```html
<dot:Repeater DataSource="{value: Rows}" RenderWrapperTag="false">        <ItemTemplate>
            <div class="row">
                <dc:DcTestControl GoToDetailAction="{command: _parent.OnGoToDetail}" />
                <dot:Repeater DataSource="{value: _parent.Columns}" RenderWrapperTag="false">
                    <ItemTemplate>
                        <div class="col-sm-4">
                            <h2>{{value: _parent}}-{{value: _this}}</h2>
                            <dc:DcTestControl GoToDetailAction="{command: _parent2.OnGoToDetail}" />
                            <bs:Button Text="Interni klik" ButtonTagName="button" Click="{command: _parent2.OnGoToDetail()}" />
                        </div>
                    </ItemTemplate>
                </dot:Repeater>
            </div>
        </ItemTemplate>
    </dot:Repeater>
```
The collection is just an int collection. Inside the light is just calling Action like this:
```html
<bs:Button Text="XXX"
           ButtonTagName="button"
           Click="{controlCommand: OnGoToDetail()}" />
```
So far I have gone through it and it is that when the dotcontrol is in the repeater, it works OK, but when the light in the repeater is in the repeater, no longer.

**Comment by exyi** [comment]
It seems to be fixed in the 2.0 release.

**Comment by quigamdev** [comment]
I consider this bug as a significant one. That's why we will release a special version with fix of this bug before releasing version 2.0 RTM.

**Comment by quigamdev** [comment]
Fix already released in 2.0 RTM.
